bokehjs/src/lib/models/markers/scatter.ts

```diff
@@ -53,6 +53,8 @@ export class ScatterView extends MarkerView {
 
     const marker: string[] = new Array(len)
     marker[index] = this._marker[index]
+    if (marker[index] === undefined)
+      return
```

Comment:
I suppose this would do the job, but the problem is really with `_render`, which should skip rendering if `_marker[index]` isn't defined. I would rather proceed with the following change:
```diff
diff --git a/bokehjs/src/lib/models/markers/scatter.ts b/bokehjs/src/lib/models/markers/scatter.ts
index 9f4950d..9a9498a 100644
--- a/bokehjs/src/lib/models/markers/scatter.ts
+++ b/bokehjs/src/lib/models/markers/scatter.ts
@@ -16,7 +16,7 @@ export class ScatterView extends MarkerView {
 
   protected _render(ctx: Context2d, indices: number[], {sx, sy, _size, _angle, _marker}: ScatterData): void {
     for (const i of indices) {
-      if (isNaN(sx[i] + sy[i] + _size[i] + _angle[i]))
+      if (isNaN(sx[i] + sy[i] + _size[i] + _angle[i]) || _marker[i] == null)
         continue
 
       const r = _size[i]/2
```

Reply:
Okay, I'll update the PR first thing in the morning.